### PR TITLE
Updated this class

### PR DESCRIPTION
**Pull request by colossus9**
Hey @jwiebalk I made this 🆒 change! Can you review?

### CODE REVIEW

**Comment by colossus9** [comment]
Hello? 👋

File: app/models/book.rb [hunk 1]
```diff
@@ -1,6 +1,6 @@
 class Book < ApplicationRecord
   validates :title, presence: true
-  #validates :year_published, numericality: { only_integer: true }
+  validates :year_published, numericality: { only_integer: true }
```
Comment (jwiebalk) [inline app/models/book.rb:3]: Totally makes sense

**Review by jwiebalk — APPROVED** [review]
LGTM 👍 :shipit:

File: app/models/book.rb (hunk 1 above)
Comment (colossus9) [reply app/models/book.rb:3]: Of course!

**Pull request merged**